docs/source/gbd2017_models/concept_models/vivarium_gates_bep/concept_model.rst

```diff
@@ -0,0 +1,353 @@
+
+
+.. _1.1:
+
+1.1 Project overview
+--------------------
+
+There is belief that benefits of BEP supplementation are currently underestimated and that targeting individual pregnant women with low BMI may be more cost beneficial. There are currently on-going Gates funded clinical trials evaluating the impact of BEP in India, Pakistan, Mali and Tanzania. This simulation project aims to estimate the cost benefits of universal or targeted offer of BEP if the anticipated improvement in infant outcomes are achieved. 
+
+
+.. _1.2:
+
+1.2 Literature review
+---------------------
+
+
+.. _2.0:
+
+2.0 Modeling aims and objectives
+++++++++++++++++++++++++++++++++
+
+To understand the cost-effectiveness of BEP and MMN supplementation for pregnant and lactating women (PLW) on child health outcomes. We will consider the costs and DALYs of baseline, and all scenarios
+
+Objective 1: Comparing the cost-effectiveness of BEP, MMN, and IFA
+This objective will compare MMN scenario vs baseline, and IFA vs MMN scenario
+
+Objective 2:  Comparing the cost-effectiveness of universal vs. targeted BEP administration
+This objective will compare scenario with targeted BEP vs scenario with universal BEP
+
+
+.. _3.0:
+
+3.0 Causal framework
+++++++++++++++++++++
+
+.. _3.1:
+
+3.1 Causal diagram
+------------------
+
+  .. image:: causal_dag_bep_scenarios.svg
+
+**Outcome (O)**:
+
+  - (1) birthweight in grams
+  - (2) LAZ score
+  - (3) WLZ score
+
+
+**Effect modifiers**:
+
+  - The effect of BEP on birthweight is differential according to maternal BMI
+
+:underline:`Current evidence scenarios`
+
+• Under the current evidence scenarios, the intervention BEP affects only birthweight. It does so differentially by maternal BMI status (Ota 2015). BMI is an effect modifier on the relationship between BEP and birthweight. 
+
+• BMI should have an effect on LAZ and WLZ scores suggested by the literature (and Gates) but we are not intervening on BMI, and so simply correlatng birthweight and LAZ / WLZ will capture the relationship between BMI and LAZ/WLZ.
+• note x represents a crude effect size because we are not intervening on BMI and we only use this effect in baseline. (ideally crude mean shift birthweight by maternal BMI)  
+• Originally Gates wanted us to model an effect size of RR=2 for BMI on LAZ and WLZ scores, which we did in our februrary model. However, since we are not intervening on BMI, we do not need to model this causal effect. We only want the baseline LAZ and WLZ scores by BMI status and this relationship should be captured by the correlation co-efficient between birthweight and LAZ and WLZ score.
```

Comment:
```suggestion
• Originally Gates wanted us to model a *causal* effect size of RR=2 for BMI on LAZ and WLZ scores, which we did in our Februrary model. However, since we are not intervening on BMI, we do not need to model this causal effect. We only want the baseline LAZ and WLZ scores by BMI status and this relationship should be captured by the correlation co-efficient between birthweight and LAZ and WLZ score, given that we also model the association between maternal BMI and birthweight.
```

Reply:
We just had an discussion with James, and he says that the correlation co-efficient between birthweight and LAZ/WLZ scores would **not** capture the relationship between BMI and LAZ/WLZ scores because of the way we are doing correlation (by the marginal distributions).  The way we had it in our february model, we _induced_ correlation between birthweight and LAZ/WLZ by means of the common cause of BMI so we should get the 'correct-ish' birthweight, LAZ, WLZ means by BMI subgroups.  

So the step forward for now is to check how well birthweight and LAZ/WLZ score correlate in our feb model (with updated LRI and LBW risk models) against the MAL-ED correlation coefficients that Chris produced. If it captures all the correlation, then good, else we might need to think again. 

Because we induced correlation by maternal BMI, we should only see the correlation structure when NOT stratified by BMI groupings. If we stratify by BMI groupings, the correlation should disappear. 

Nicole will run an interactive sim (with the help of Kjell) and produce individual level data at a time-point for those aged 6 months+, stratified by age (can collapse over sex as there is no effect of sex on bw/laz/wlz) and plot birthweight with laz and birweight with wlz to get the correlation coefficients and compare them with the MAL-ED data. 

@collijk @alibow @KjellSwedin @alibow @aflaxman Does this make sense? Let me know if I've missed anything!

Reply:
Hmm, I am still confused by this. I think I am not understanding how we are using marginal distributions in a way that would not capture the association between maternal BMI and child stunting/wasting. Could you please explain that some more?

Reply:
![image](https://user-images.githubusercontent.com/57811075/87089841-e31c9d00-c1eb-11ea-98e3-583ffffc530e.png)

I think...it's cuz the low BMI and normal BMI mothers have the different distributions with different means. so the low BMI mothers will draw from a birthweight distribution with lower means, and the high BMI mothers will draw from a birthweight dsitrubion with higher means, but the LAZ/WLZ distribution is the same. 

Now I'm wondering why we can't just draw at the lower end of the birthweight distribution (if low BMI) and higher end of the distribution (for high BMI mothers). As in we have one distribution for birthweight and just draw from different areas of the same distribution depending that is determined by the birthweight mean difference among BMI subgroups...?

Reply:
so even if we have individual birth weights for each simulant (which will be based on their maternal BMI status), we cannot sample corresponding correlated WLZ and LAZ scores using the same methodology as above based on their existing birth weight?

Reply:
![BEP_correlation_vis](https://user-images.githubusercontent.com/57811075/88135039-f724af00-cb9a-11ea-9a6e-1194664ac41e.jpg)

this has not yet been resolved and hoping to chat with Abie/James tomorrow (july 22 2020). I'm uploading an image of my understanding of it for the record.